yang/ietf-network-inventory.yang

```diff
@@ -198,14 +198,22 @@ module ietf-network-inventory {
         config false;
         description
           "The vendor-specific revision string of the software
-           module.";
+           module when not implicitly defined as part of the name of
+           the software module.";
```

Comment:
Are you sure that also the SW revision could be part of the name ? My understanding was that was just for patch revision. This is from Mahesh: "...In cases such as ours (Arrcus, and I have seen this with other vendors also), the software patch version/rev is part of the software name (or rev) and is not a separate field"

Reply:
My understanding of "software patch version/rev is part of the software name (or rev)" is that: the software patch version/rev is part of the software name (which includes also the software revision) or the software patch version/rev is part of the software revision

Reply:
Please see the follow-up comment on https://github.com/ietf-ivy-wg/network-inventory-yang/issues/128#issuecomment-3377067877